t/api/04-jobs.t

```diff
@@ -1206,10 +1206,10 @@ subtest 'create job failed when PUBLISH_HDD_1 is invalid' => sub {
 
 subtest 'show job modules execution time' => sub {
     my %modules_execution_time = (
-        aplay              => 146,
-        consoletest_finish => 164,
-        gnucash            => 187,
-        installer_timezone => 34
+        aplay              => '2 m 26 s',
```

Comment:
Wouldn't `2m 26s` look better without the whitespace?

Reply:
Seriously? I find the white space more clean and consistent. You would have a white space if you'd spell out the abbreviations for minutes and seconds.

Reply:
And common pages like Wikipedia also put a white space between the number and its unit, e.g. https://en.wikipedia.org/wiki/Metre

Reply:
Looks like we need a vote. 😁

Reply:
According the result of voting, I remove the space :)

Reply:
Not that this means not following conventions by the SI: https://en.wikipedia.org/wiki/Space_(punctuation)#Unit_symbols_and_numbers, https://www.bipm.org/en/publications/si-brochure/section5-3.html

Reply:
I just think this with no space look nicer.  And there is same situation in the OS, such as
```
liuxiaojing@linux-90dc:~> time ls
...
real    0m0.006s
user    0m0.002s
sys     0m0.004s
``` 
If there is no severe effect, maybe we could remove the space.